conf/timeseries-dev-source.conf

```diff
@@ -1,5 +1,5 @@
     dataset = "prometheus"
-    num-shards = 4
+    num-shards = 8
```

Comment:
Tanvi have you ran through all of the local testing in the README to ensure queries still work?  Please also test the path involving local gateway also.  This change changes a lot of local standalone scenarios and needs to be tested thoroughly

Reply:
Sorry I checked this in by mistake